specfile/utils.py

```diff
@@ -1,9 +1,54 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
 
+import re
 import urllib.parse
+from dataclasses import dataclass
 from pathlib import Path
 
+from specfile.exceptions import SpecfileException
+
+
+@dataclass(kw_only=True)
+class EVR:
+    epoch: int = 0
```

Comment:
please document what EVR stands for so it's clear for everyone who's not familiar with this shortcut